iam-role-policy-changes-check/check/pr.go

```diff
@@ -0,0 +1,61 @@
+// Check is used to perform bool checks on whether a PR
+// contain potential IAM Role and Policy changes/additions 
+// to alert a reviewer to the above content
+package check
```

Comment:
Can you change the package name to something specific to the iam. This repo already have a package check under "approve-doc-preview"